registry/debug-adapters.yml

```diff
@@ -16,6 +16,42 @@ debug-adapters:
       protocol: swd
       clock: 10000000
 
+  - name: "MCU-Link@pyOCD"
+    alias-name: ["MCU-Link"]                  # alternative names that map to this debug adapter
+    template: CMSIS-DAP-pyOCD.adapter.json    # template file (initally same as CMSIS-DAP@pyOCD)
+    gdbserver:                                # add gdbserver: node under debugger: in cbuild-run.yml
+    defaults:                                 # default values to use when nowhere specified
+      port: 3333                              # default value of first gdbserver port
+      protocol: swd
+      clock: 10000000
+
+  - name: "Nu-Link@pyOCD"
+    alias-name: ["Nu-Link", "Nu-Link-Pro"]    # alternative names that map to this debug adapter
+    template: CMSIS-DAP-pyOCD.adapter.json    # template file (initally same as CMSIS-DAP@pyOCD)
+    gdbserver:                                # add gdbserver: node under debugger: in cbuild-run.yml
+    defaults:                                 # default values to use when nowhere specified
+      port: 3333                              # default value of first gdbserver port
+      protocol: swd
+      clock: 10000000
+
+  - name: "PICkit@pyOCD"
+    alias-name: ["PICkit Basic"]              # alternative names that map to this debug adapter
```

Comment:
This is the first name with a space.

Reply:
Are spaces not allowed?

Reply:
JLink Server uses a space as well. Discussed with Jonatan, it is allowed.